src/main/groovy/com/f2prateek/javafmt/AndroidJavaFmtExtension.groovy

```diff
@@ -0,0 +1,5 @@
+package com.f2prateek.javafmt;
+
+class AndroidJavaFmtExtension {
+  boolean aospStyle = false
```

Comment:
hmmm.. is that ever possible?

1. We use the Google enum as is. In this case there's no chance that an incorrect one can be applied.

2. We create our own enum:
```
enum Style {
AOSP(JavaFormatterOptions.Style.AOSP)
Google(JavaFormatterOptions.Style.Google)
}
```
In this case it's never possible to accidentally supply a style that java-format doesn't support. It's only possible if Google removes one of the existing options.